File: app/src/main/java/com/tangbba/simplegithubforjava/api/model/RepoSearchResponse.java
package com.tangbba.simplegithubforjava.api.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RepoSearchResponse {

    @SerializedName("total_count")
    private int mTotalCount;
    @SerializedName("items")
    private List<GithubRepo> mItems;

    public RepoSearchResponse(int totalCount, List<GithubRepo> items) {
        mTotalCount = totalCount;
        mItems = items;
    }

    public int getTotalCount() {
        return mTotalCount;
    }

    public void setTotalCount(int totalCount) {
        mTotalCount = totalCount;
    }

    public List<GithubRepo> getItems() {
        return mItems;
    }

    public void setItems(List<GithubRepo> items) {
        mItems = items;
    }
}

File: app/src/main/kotlin/com/tangbba/simplegithubforjava/api/model/RepoSearchResponse.kt
package com.tangbba.simplegithubforjava.api.model

import com.google.gson.annotations.SerializedName

class RepoSearchResponse(@field:SerializedName("total_count") val totalCount: Int,
                         @field:SerializedName("items") val items: List<GithubRepo>)
